### ISSUE TITLE
FEATURE: For attributes that include a LOT of text, need a way to limit display in the table

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#288]
If an attribute has a paragraph or more of text, the nodes table becomes effectively unusable. It'd be nice to either / or both:

1) be able to hide that column in table view (maybe via a checkbox?)
2) truncate text beyond a certain length and either require the node info panel to see it, or add an. inline "more" button

Assuming this is likely a "next grant" kind of feature but wanted to note it while it was fresh.

**Comment by jdanish** [comment]
@benloh we are using markdown-it right?  If adding markdown-it-expandable as a plugin would not be an issue, it'd be a nice way to address this with minimal work on our end. I think.

**Comment by benloh** [comment]
> adding markdown-it-expandable as a plugin

If you're talking about this plugin -- https://www.npmjs.com/package/markdown-it-expandable -- I'm not sure it'll help.  Your text has to be entered like this:

```
+++ Click me!
Hidden text
+++
```

This might work for help text or other researcher-authored text, but it won't work for kids.

**Comment by jdanish** [comment]
Yeah, I was thinking as a short-term solution because currently the longest text is something we add, not kids. But long-term, it'd be much cleaner if it was basically the ability to shrink table rows and then expand them if certain fields were too big. I guess let's wait until we can tackle it in a smart way?

**Comment by benloh** [comment]
This is currently addressed with the tool tip.  Though column selection is not implemented.
